### PR TITLE
NMS-15799: k8s: docs: Review and update the documentation

### PR DESCRIPTION
**Pull request by Bonrob2**
Updated Helm Charts docs to remove references to "UI instances", update the text around kar-container, and other changes requested in the ticket.

### CODE REVIEW

File: docs/modules/about/pages/about.adoc [hunk 1]
```diff
@@ -54,16 +54,15 @@ You could build your images and move the logic from the scripts executed via `in
 The scripts configure only a certain number of things.
 Each deployment would likely need additional configuration, which is the main reason for using a Persistent Volume Claim (PVC) for the configuration directory of the Core OpenNMS instance.
 
-We must place the core configuration on a PVC configured as `ReadWriteMany` to allow the use of independent UI servers so that the Core can make changes and the UI instances can read from them.
+We must place the core configuration on a PVC configured as `ReadWriteMany` to allow the use of independent UI servers so that the Core can make changes.
```
Comment (mershad-manesh) [inline docs/modules/about/pages/about.adoc:57]: I think we can safely remove this line as we are not using `ReadWriteMany` and we have dropped support for UI servers.

Lets confirm with @deejgregor, to make sure it makes sense to drop this line.

File: docs/modules/about/pages/about.adoc [hunk 2]
```diff
@@ -54,16 +54,15 @@ You could build your images and move the logic from the scripts executed via `in
...
 One advantage of configuring that volume is allowing backups and access to the files without accessing the OpenNMS instances running in Kubernetes.
 
-
 === Time series databases
 
 Similarly, when using RRDtool instead of Newts/Cassandra or Cortex, a shared volume with `ReadWriteMany` is required for the same reasons (the Core would be writing to it, and the UI servers would be reading from it).
```
Comment (mershad-manesh) [inline docs/modules/about/pages/about.adoc:62]: I think the same comment as above applies to this one! 

We should confirm this.

File: docs/modules/installation/pages/introduction.adoc [hunk 3]
```diff
@@ -45,11 +45,11 @@ When using cloud resources, make sure you have https://learn.microsoft.com/en-us
 It must not contain special characters and must follow FQDN restrictions.
 * A single instance of OpenNMS Core (backend) for centralized monitoring running ALEC in standalone mode (if enabled).
 ** OpenNMS does not support distributed mode, meaning the `StatefulSet` cannot have more than one replica.
-* A shared volume for the RRD files, mounted as read-write on the Core instance, and as read-only on the UI instances if applicable.
-* A shared volume for the core configuration files, mounted as read-only on the UI instances if applicable.
+* A shared volume for the RRD files, mounted as read-write on the Core instance.
+* A shared volume for the core configuration files.
 ** Allows for sharing configuration across all the OpenNMS instances (for example, `users.xml`, and `groups.xml`).
 * Multiple instances of Grafana (frontend), using PostgreSQL as the backend, pointing to the OpenNMS UI service when available.
```
Comment (deejgregor) [inline docs/modules/installation/pages/introduction.adoc:51]:
```suggestion
* Multiple instances of Grafana (frontend), using PostgreSQL as the backend, pointing to the OpenNMS UI.
```

We no longer have a separate UI service.

**Comment by deejgregor** [comment]
There's a block in the introduction that should go:
```
* Multiple instances of read-only OpenNMS UI (frontend).
** Must be stateless (unconfigurable).
** The `Deployment` must work with multiple replicas.
** Any configuration change goes to the core server.
```

File: docs/modules/about/pages/about.adoc (hunk 1 above)
Comment (Bonrob2) [inline docs/modules/about/pages/about.adoc:57]:
```suggestion

```

File: docs/modules/about/pages/about.adoc (hunk 2 above)
Comment (Bonrob2) [inline docs/modules/about/pages/about.adoc:62]:
```suggestion

```

File: docs/modules/about/pages/about.adoc [hunk 4]
```diff
@@ -54,16 +54,13 @@ You could build your images and move the logic from the scripts executed via `in
...
 Each deployment would likely need additional configuration, which is the main reason for using a Persistent Volume Claim (PVC) for the configuration directory of the Core OpenNMS instance.
 
-We must place the core configuration on a PVC configured as `ReadWriteMany` to allow the use of independent UI servers so that the Core can make changes and the UI instances can read from them.
 One advantage of configuring that volume is allowing backups and access to the files without accessing the OpenNMS instances running in Kubernetes.
 
-
 === Time series databases
 
-Similarly, when using RRDtool instead of Newts/Cassandra or Cortex, a shared volume with `ReadWriteMany` is required for the same reasons (the Core would be writing to it, and the UI servers would be reading from it).
 Additionally, when switching strategies and migration are required, you could work outside Kubernetes.
```
Comment (deejgregor) [inline docs/modules/about/pages/about.adoc:61]: I think we can simplify these this entire section:
```suggestion
Multiple time series strategies are supported and you can switch from one to another as needed. Please note when switching strategies existing data is not migrated.
```

**Review by deejgregor — DISMISSED** [review]

**Review by mershad-manesh — APPROVED** [review]

**Pull request merged**